### DISCUSSION TITLE
No registered env with id: KukaBulletEnv-v0

### DISCUSSION THREAD

**Discussion by Jingjinganhao** [bulletphysics/bullet3#3644] [General]
Hi, I have trained model by running
python -m pybullet_envs.agents.train_ppo --config=pybullet_kuka_grasping --logdir=kuka
but when I run 
python -m pybullet_envs.agents.visualize_ppo  --logdir=kuka/<timestampname> --outdir=kuka_video1
It shows
No registered env with id: KukaBulletEnv-v0

**Comment by wenshuaizhao** [comment]
> Hi, I have trained model by running
> python -m pybullet_envs.agents.train_ppo --config=pybullet_kuka_grasping --logdir=kuka
> but when I run
> python -m pybullet_envs.agents.visualize_ppo --logdir=kuka/ --outdir=kuka_video1
> It shows
> No registered env with id: KukaBulletEnv-v0

Hi, did you solve such a problem? I just encountered the same
